app/scripts/modules/ecs/src/serverGroup/configure/wizard/taskDefinition/TaskDefinition.tsx

```diff
@@ -342,6 +352,15 @@ export class TaskDefinition extends React.Component<ITaskDefinitionProps, ITaskD
             />
           </div>
         </div>
+
+        <StageConfigField groupClassName="form-group evaluateTaskDef" label="Expression Evaluation" helpKey="ecs.evaluateExpression" labelColumns={5} fieldColumns={7} >
+          <CheckboxInput
+            checked={command.evaluateTaskDefinitionArtifactExpressions === true}
+            onChange={() =>  { updateEvaluateTaskDefArtifactFlag(); }}
+            text="Evaluate Task Definition Artifact"
```

Comment:
be more explicit here: "Evaluate SpEL expressions in artifact" (I think we can omit "task definition" b/c that's the section of the wizard we're in here)

Reply:
make sense